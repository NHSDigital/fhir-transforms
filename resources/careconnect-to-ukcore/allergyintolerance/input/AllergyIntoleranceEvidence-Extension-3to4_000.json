{
  "resourceType": "Encounter",
  "extension": [
    {
      "url": "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-Evidence-1",
	  "valueReference": {
	    "reference": "DiagnosticReport/3BF93498-ABCF-4326-B5F4-071EDB142D60"
	  }
    }
  ]
}
