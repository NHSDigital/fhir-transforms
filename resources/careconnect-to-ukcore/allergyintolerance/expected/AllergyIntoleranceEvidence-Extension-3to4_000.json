{
  "resourceType": "AllergyIntolerance",
  "extension": [
    {
      "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-Evidence",
      "valueReference": {
        "reference": "DiagnosticReport/3BF93498-ABCF-4326-B5F4-071EDB142D60"
      }
    }
  ]
}